### DISCUSSION TITLE
OpenTalk

### DISCUSSION THREAD

**Discussion by bbbhltz** [qutebrowser/qutebrowser#7491] [General] [answered]
Hi,

I didn't want to add to the Issues with another "this site doesn't work" thing, but I have recently been given access to OpenTalk. I cannot seem to get it to work with qutebrowser. There are demo chatrooms here https://opentalk.eu/en/demo that you can test out. If it works for anybody else, could you let me know and I will try and I will dig a little deeper.

The error I get is the boilerplate

> Your browser is not supported.
> Please use the latest version of Firefox, or Chrome. If you have further difficulties, check if your browser is running in compatibility or incognito mode. Disable them and try again here.
> 
> Your Opentalk team

**Comment by mschilli87** [comment, marked as answer]
What version are you running? Did you try the Qt6 branch? Can you reproduce this with `--temp-basedir`?

**Reply by The-Compiler** [reply]
I can reproduce, and indeed it works fine with Qt 6. I can't seem to find out what the culprit is, but probably no easy way to make it work with Qt 5 anyways.

**Reply by bbbhltz** [reply]
Well, I guess I don't need to wait until I get home from work. Thanks. I guess I will be moving on to Qt 6 if I do decide to use that service.
